app/controllers/HomeController.scala

```diff
@@ -5,16 +5,17 @@ import javax.inject._
 import com.typesafe.scalalogging.LazyLogging
 import play.api.mvc._
 
-import scala.concurrent.ExecutionContext.Implicits.global
-import scala.concurrent.Future
+import scala.concurrent.{ExecutionContext, Future}
```

Comment:
Accidentally deleted @rishabh9 comment, sorry!

He asked: How did you infer this? @scozv